Userland/Libraries/LibJS/Runtime/ArrayConstructor.cpp

```diff
@@ -30,6 +36,7 @@ ThrowCompletionOr<void> ArrayConstructor::initialize(Realm& realm)
     define_direct_property(vm.names.prototype, realm.intrinsics().array_prototype(), 0);
 
     u8 attr = Attribute::Writable | Attribute::Configurable;
+    define_native_function(realm, vm.names.fromAsync, from_async, 1, attr);
```

Comment:
This goes after `from`

Reply:
![image](https://github.com/SerenityOS/serenity/assets/35911232/3bbaeb2c-c090-4454-a65e-0bc2b23b784c)

Just out of interest, where does that convention came from? I ordered it this way based on this editors note

Reply:
yep, spec always orders alphabetically, so while the section numbers are out of order in this case, this will be where they place it when merged in the main spec :)

Reply:
Thanks, that makes sense!

Reply:
(the note is very likely wrong FWIW, it shouldn't end up before `from`)